src/PageItem.tsx

```diff
@@ -13,6 +13,8 @@ export interface PageItemProps
   active?: boolean;
   activeLabel?: string;
   href?: string;
+  innerStyle?: React.CSSProperties;
```

Comment:
Can you rename these to `linkStyle` and `linkClassName`?  It's more descriptive of where the prop is applied.  Also please add tests for the two props